src/mscorlib/shared/System/Globalization/CalendarData.Unix.cs

```diff
@@ -257,6 +267,15 @@ private static bool EnumMonthNames(string localeName, CalendarId calendarId, Cal
                     callbackContext.Results.Add(string.Empty);
                 }
 
+                if (callbackContext.Results.Count > 13)
+                {
```

Comment:
Add `Debug.Assert(callbackContext.Results.Count == 14 && calendarId == CalendarId.HEBREW)` ?

Reply:
In the future we can get calendars which can have 13 months too. so I prefer not putting this assert.

Reply:
This path is only taken for 14 months or more...

I was just wondering whether assert like that would help us if some other calendar needs the special treatment like Hebrew.

Reply:
I'll add the assert. I think it make sense to be conscious about any other similar behaving calendar